### ISSUE TITLE
Broken stac-spec links

### ISSUE THREAD

**Issue by duckontheweb** [radiantearth/stac-api-spec#114]
Links to entities defined in the [radiantearth/stac-spec](https://github.com/radiantearth/stac-spec) repo are set up as relative links within the `stac-spec` folder. Since this is actually a submodule these links don't work as expected and should maybe be updated to be absolute URLs.

I can work on a PR to fix this if it's not already underway.

**Comment by cholmes** [comment]
So the problem with absolute links is keeping them up to date and properly pinned to the right STAC release. We don't want to just go to master since that may change. The submodule seemed to give us control and easy updates. 

Is it not working even when you've checked the sub-module out correctly? Or are you just saying that people shouldn't have to check out the sub-module to have links work right?

**Comment by duckontheweb** [comment]
The problem I'm seeing is with the README hosted on GitHub. Someone going to https://github.com/radiantearth/stac-api-spec and clicking on the link to one of those objects (e.g. the `Catalog` link in the "About" section of the main README) gets a 404 response.

**Comment by cholmes** [comment]
Fair point, and yeah, that seems less than ideal. I wonder if they used to work, as it seems like github has changed some of their links around recently. 

A PR to fix would be great, and then perhaps also file a ticket to spur us to get a robust way to update the links when we release.

**Comment by duckontheweb** [comment]
Actually, using a [`git subtree`](https://www.atlassian.com/git/tutorials/git-subtree) instead of a `git submodule` might solve our problem here. Subtrees actually copy the child repo into the parent, which would keep links intact (versus submodules, which just point to a commit on the child repo). It requires `git>=v1.7`, but that version is years old, so it shouldn't be a problem.

**Comment by m-mohr** [comment]
@duckontheweb I think I'd be in favor of subtree than absolute links, if we can make it that PRs to the subtree get rejected or so.

**Comment by cholmes** [comment]
+1 on the subtree. To me that's ok even if we can't get automatic rejection of the PR's, since we'll soon turn on the requirement of 2 reviewers as we have in stac-spec. Reviewers would know that edits to stac-spec aren't supposed to be done here.

**Comment by duckontheweb** [comment]
> @duckontheweb did your PR resolve this issue?

Yes, you can close this. I'll also close #115, which is now no longer necessary.
